### PR TITLE
[PTX-1840] Send kill -SIGUSR1 <px_process_id> while collecting diags

### PR DESCRIPTION
**Pull request by vinayakshnd**

### CODE REVIEW

File: drivers/volume/portworx/portworx.go [hunk 1]
```diff
@@ -1572,18 +1572,29 @@ func (d *portworx) CollectDiags(n node.Node) error {
 		return err
 	}
 
+	opts := node.ConnectionOpts{
```
Comment (nikolaypopov) [inline drivers/volume/portworx/portworx.go]: Why did you remove Sudo: true from here, it was required for collecting diags on some of the systems. It was added in PTX-1825

Comment (vinayakshnd) [reply drivers/volume/portworx/portworx.go]: @nikolaypopov , Added "Sudo: true" back.

**Comment by vinayakshnd** [comment]
@nikolaypopov  addressed review comments
